resources/content/schema/base/service.json

```diff
@@ -49,14 +49,11 @@
                 "scheduleUpdate" : true
             }
         },
-        "previousLaunchConfig": {
-            "type": "launchConfig",
+        "domainName":{
+            "type": "string",
+            "required": false,
             "nullable": true
         },
-        "previousSecondaryLaunchConfigs": {
```

Comment:
unrelated. Removed it because its not used anywhere (was left around from some other PR, forgot to clean them up)